root/etc/e-smith/templates/etc/sudoers.d/50_servermanager_perms/10base

```diff
@@ -0,0 +1,11 @@
+#
+# Cmnd_Alias association to group permission
+#
+
+# all users
+%locals ALL=NOPASSWD: NSAPI_PUBLIC
+%domain\ users ALL=NOPASSWD: NSAPI_PUBLIC
+
+# server-manager does not require a tty
+Defaults:%locals !requiretty
```

Comment:
Is it possible to associate the `!requiretty` flag to a CmdAlias list instead of a group name?

Indeed, I see the `!requiretty` a property of the command and not of the user/group.

Reply:
https://serverfault.com/questions/246135/is-it-possible-to-use-sudo-with-requiretty-for-a-specific-command